Library/Homebrew/formula.rb

```diff
@@ -1487,12 +1488,29 @@ def recursive_requirements(&block)
   # Returns a list of Dependency objects that are required at runtime.
   # @private
   def runtime_dependencies
```

Comment:
I was thinking: given this is going to change the output of `runtime_dependencies` again quite drastically: I think it would be worth having the `runtime_dependencies` in the Tab be versioned. This means we don't need to do something like `parsed_homebrew_version < "1.1.6"` (in `tab.rb`) again in order to make sure these tabs have what we expect/need to rely on them.

Reply:
I've changed my mind on this given https://github.com/Homebrew/brew/pull/3979 so: ignore it.